php/meterpreter/meterpreter.php

```diff
@@ -42,8 +42,27 @@ function register_command($c, $i) {
   }
 }
 
+# Debugging payload definitions
+define("DEBUGGING", false);
```

Comment:
Hm; I wonder if this would conflict with _other_ debugging flags in a running PHP application 🤔

Reply:
I've set this to `MY_DEBUGGING` instead to follow the pattern of `my_logfile` and `my_print`.